glibc.yaml

```diff
@@ -436,6 +437,11 @@ subpackages:
             && mv "${{targets.destdir}}"/usr/lib/locale/${{range.key}} "${{targets.subpkgdir}}"/usr/lib/locale/
 
 advisories:
+  CVE-2010-4756:
+    - timestamp: 2023-03-06T12:47:28.379717-05:00
+      status: not_affected
+      justification: vulnerable_code_not_present
```

Comment:
I don't think this is a correct justification for this specific case (though I admit it is the closest one).  There is a vulnerability, but the vulnerability is in the consumer of the API in this case.  Maybe we should add additional justification types to OpenVEX?

Reply:
This has come up before: https://github.com/wolfi-dev/os/blob/main/flex.yaml#L75

tl;dr: Yes there should be a new enum value here, but that's a larger issue. We have a few of these already, and we'll want to come back and address them all once there's a new status.

I'd vote for not letting that block this PR.

Reply:
Sounds good, just as long as we have an issue tracking adding an additional enum.